src/topp/OpenSwathWorkflow.cpp

```diff
@@ -921,7 +923,7 @@ class TOPPOpenSwathWorkflow
     ///////////////////////////////////
     FeatureMap out_featureFile;
     OpenSwathTSVWriter tsvwriter(out_tsv, file_list[0], use_ms1_traces, sonar); // only active if filename not empty
-    OpenSwathOSWWriter oswwriter(out_osw, run_id, file_list[0], use_ms1_traces, sonar, enable_uis_scoring); // only active if filename not empty
+    OpenSwathOSWWriter oswwriter(out_osw, run_id, file_list[0], use_ms1_traces, sonar, compute_peak_shape_metrics, enable_uis_scoring); // only active if filename not empty
```

Comment:
there should  be a test which covers the new functionality!

Reply:
tests are a bit difficult because I do not know if the output for .osw can be really compared. I can add a test with compute_peak_shape_metrics to the black-box tests but I do not think there are currently any tests for OpenSwathOSWWriter

Reply:
That would be great